### PR TITLE
Adding fingerprint_hash_type to known host example.

### PR DESCRIPTION
**Pull request by nixjdm**
Since 2017.7.0 [the existing example](https://docs.saltstack.com/en/latest/ref/states/all/salt.states.ssh_known_hosts.html) would not have worked without this change.

### What does this PR do?
Updates documentation to conform to existing behavior, changed in 2017.7.0

### What issues does this PR fix or reference?
None. It's a simple docs update and I couldn't find a related ticket.

### Tests written?

N/A

### CODE REVIEW

File: salt/states/ssh_known_hosts.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
         - present
         - user: root
         - fingerprint: 16:27:ac:a5:76:28:2d:36:63:1b:56:4d:eb:df:a6:48
+        - fingerprint_hash_type: md5
```
Comment (cachedout) [inline salt/states/ssh_known_hosts.py]: This should be `sha256` though.

**Review by cachedout — CHANGES_REQUESTED** [review]

File: salt/states/ssh_known_hosts.py (hunk 1 above)
Comment (nixjdm) [inline salt/states/ssh_known_hosts.py]: The current line 14 here is [the hexed md5 format of GitHub's fingerprint](https://help.github.com/articles/github-s-ssh-key-fingerprints/).

They could be two different examples. There currently isn't a sha256 example in the docs anywhere that I've seen.

Comment (cachedout) [inline salt/states/ssh_known_hosts.py]: Then we need to change the fingerprint example as well. :]

Comment (cachedout) [inline salt/states/ssh_known_hosts.py]: @nixjdm Did you want to change the examples to SHA-256 and away from MD5? If not, we can have somebody else do it and then come back to this PR after that is done.

Comment (nixjdm) [inline salt/states/ssh_known_hosts.py]: I'm fine having only a sha256 example. I'll give it a shot. I'll try to get my own code using the sha256 hast type and update this if I get that working. Give me a couple days, please.

Comment (nixjdm) [inline salt/states/ssh_known_hosts.py]: @cachedout I haven't gotten sha256 to work for me, so I'll not add that example and leave this md5 one.

I tried with the github hash of `SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8`, and without the prefix as `nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8`, with and without a line reading `- fingerprint_hash_type: sha256`. If I'm simply doing something wrong, I'd appreciate some else fixing this bit of documentation so I can use sha256 myself. In the meantime I'll have to stick with md5.

**Comment by cachedout** [comment]
Go Go Jenkins!

**Pull request merged**